front/src/modules/timesStops/hooks/useOutputTableData.ts

```diff
@@ -89,11 +90,15 @@ function useOutputTableData(
             selectedTrainSchedule,
             pathStepId
           );
+          const calculatedArrival = secToHoursString(opPoint.time, true);
+          const arrivalValuesAreClose =
+            Math.abs(compareTimeString(scheduleData.arrival, calculatedArrival)) <= 1000;
```

Comment:
How about auto-documented code with the following proposition (without it, it looks like a non-explained arbitrary value). I'm open to any other naming, I just didn't came up with a good idea.

```suggestion
          const MAXIMAL_ROUNDING_ERROR_IN_MS = 1000;
          const arrivalValuesAreClose =
            Math.abs(compareTimeString(scheduleData.arrival, calculatedArrival)) <= 1000;
```

Reply:
added a named constant